libs/designsystem/src/lib/testing/test-helper.ts

```diff
@@ -26,10 +30,15 @@ export class TestHelper {
     });
   }
 
+  /** Checks for the Ionic Web Component being hydrated, ie. the Shadow DOM is ready for query */
```

Comment:
Looks like a copy of the documentation for the `whenHasCssClass`-function, I'm guessing this function doesn't have the same behaviour?

Reply:
Actually it was the other way around ;)